.travis.yml

```diff
@@ -10,6 +10,8 @@ bundler_args: --without production
 before_script:
   - mkdir -p tmp/Dataload/EndowRpt
   - mkdir -p tmp/Dataload/BookplateMerge/Batches/Queue
+  - sudo mkdir -p /symphony/Dataload/UserloadRerun
+  - sudo mkdir -p /symphony/Dataload/ILLiadUserExport
   - sudo mkdir -p /symphony/Libsys_webforms/sal3_batch_requests_batch/bc_file
   - sudo chmod -R a+w /symphony
```

Comment:
Can these be added to `tmp/Dataload` instead of `/symphony/Dataload`? I find tests that require files in these directories fail when running them locally.

Reply:
What about `symphony/Libsys_webforms/sal3_batch_requests_batch/bc_file`?

Reply:
Ok, let's fix this in a different PR.

Reply:
See https://github.com/sul-dlss/libsys-webforms/pull/new/bc-file-location